packages/calcite-components/src/components/input-message/input-message.scss

```diff
@@ -3,34 +3,39 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-input-message-spacing-value: The top margin spacing above the component.
+ * @prop --calcite-input-message-spacing-value: [Deprecated] Use `--calcite-input-message-spacing`.The top margin spacing above the component.
+ * @prop --calcite-input-message-spacing: Specifies the top margin space above the component.
+ * @prop --calcite-input-message-icon-color: Specifies the component's icon color.
  */
 
 :host {
   @apply text-color-1 transition-default box-border flex h-auto w-full items-center font-medium opacity-100;
-  --calcite-input-message-spacing-value: theme("spacing.1");
-  margin-block-start: var(--calcite-input-message-spacing-value);
+
+  margin-block-start: var(
+    --calcite-input-message-spacing,
+    var(--calcite-input-message-spacing-value, var(--calcite-spacing-xxs))
+  );
 }
 
 .calcite-input-message-icon {
   @apply transition-default pointer-events-none inline-flex flex-shrink-0;
-  margin-inline-end: theme("margin.2");
+  margin-inline-end: var(--calcite-spacing-sm);
 }
 
 :host([status="invalid"]) .calcite-input-message-icon {
-  color: var(--calcite-color-status-danger);
+  color: var(--calcite-input-message-icon-color, var(--calcite-color-status-danger));
```

Comment:
These could probably _also_ support using `--calcite-icon-color` - currently that doesn't work to override.